standalone_apis.go

```diff
@@ -250,6 +255,11 @@ func buildQuery(resourceType string, resourceIDs []string, subjects []string, gr
 		queryPartForSubjectOrGroups["$or"] = append(tempQuery, groupsQuery)
 	}
 
+	if resourceType == "" {
+		query := queryPartForSubjectOrGroups
+		return json.Marshal(query)
```

Comment:
no need for the query definition above

```suggestion
		return json.Marshal(queryPartForSubjectOrGroups)
```

Reply:
removed useless query definition